hubblestack/extmods/utils/s3.py

```diff
@@ -203,6 +204,11 @@ def query(key, keyid, method='GET', params=None, headers=None,
             err_code = 'http-{0}'.format(result.status_code)
             err_msg = err_text
 
+    if os.environ.get('MOCK_SLOW_DOWN'):
+        result.status_code = 503
+        err_code = 'SlowDown'
+        err_msg = 'slow down you jerks'
```

Comment:
I don't think this is an appropriate error message, could we change it to something more informative?

Reply:
Yeah, that seems reasonable. Although it's worth nothing, this message is only for manual testing and it's always a fatal condition in the s3fs.